### PR TITLE
__str__ and _repr_pretty_ for Comment

### PR DESCRIPTION
**Pull request by bonaime**
### What does this PR do?
`__str__` and `_repr_pretty_` for Comment

### PR Checklist
- [X] Correct base branch selected? `master` for new features, `maintenance_...` for bug fixes
- [X] This PR is not directly related to an existing issue (which has no PR yet).
- [ ] All tests still pass.
- [x] Any new features or fixed regressions are be covered via new tests.
- [x] Significant changes have been added to `CHANGELOG.txt` .

### CODE REVIEW

File: CHANGELOG.txt [hunk 1]
```diff
@@ -40,7 +40,8 @@ master:
       information.
     * Add explicit warnings regarding unsupported sections of Nordic files.
     * Fix mapping of magnitude-types between MS to S and Ms to s.
-
+ - obspy.core.inventory/util:
+    * __str__ and _repr_pretty_ for Comment (see #2115)
```
Comment (megies) [inline CHANGELOG.txt]: Please include your bullet point under "- obspy.core:" section at top of list..

File: obspy/core/tests/test_util_types.py [hunk 2]
```diff
@@ -1,14 +1,17 @@
 # -*- coding: utf-8 -*-
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from future.builtins import *  # NOQA
-
```
Comment (megies) [inline obspy/core/tests/test_util_types.py]: Please keep order of imports.. future.builtins must come right after the `__future__` import

File: obspy/core/tests/test_util_types.py [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
 
+from future.builtins import *  # NOQA
+
 from obspy.core.util import (ComplexWithUncertainties, Enum,
                              FloatWithUncertainties)
 from obspy.core.util.obspy_types import (FloatWithUncertaintiesAndUnit)
 
+from obspy.core.inventory  import Comment
+
+from obspy import UTCDateTime
```
Comment (megies) [inline obspy/core/tests/test_util_types.py]: please stick to our alphabetical ordering scheme:

```
from obspy import UTCDateTime
from obspy.core.inventory  import Comment
from obspy.core.util import (ComplexWithUncertainties, Enum,
                             FloatWithUncertainties)
from obspy.core.util.obspy_types import (FloatWithUncertaintiesAndUnit)
```

**Comment by megies** [comment]
Ideally you should do a rebase  to get rid of that merge commit. But if you don't know how to do it right away, just leave as is.

**Comment by bonaime** [comment]
I don't know how to rebase...

File: obspy/core/tests/test_util_types.py (hunk 2 above)
Comment (krischer) [reply obspy/core/tests/test_util_types.py]: This still needs to be done.

File: obspy/core/tests/test_util_types.py [hunk 4]
```diff
@@ -116,6 +120,21 @@ def test_floating_point_types_are_indempotent(self):
         self.assertEqual(f.upper_uncertainty, 1.5)
         self.assertEqual(f.unit, "AB")
 
+    def test_commnent_str(self):
```
Comment (krischer) [inline obspy/core/tests/test_util_types.py]: Type in the method name.

**Comment by megies** [comment]
Rebased, cleanup and force-pushed

**Pull request merged**